traitlets/traitlets.py

```diff
@@ -949,7 +949,24 @@ def __init__(self, *args, **kwargs):
                 else:
                     # passthrough args that don't set traits to super
                     super_kwargs[key] = value
-        super(HasTraits, self).__init__(*super_args, **super_kwargs)
+        try:
+            super(HasTraits, self).__init__(*super_args, **super_kwargs)
+        except TypeError as e:
+            arg_s_list = [ repr(arg) for arg in super_args ]
+            for k, v in kwargs.items():
```

Comment:
This should be super_kwargs instead of kwargs (sorry, didn't spot this until after merging)